docs/README_RECON.md

```diff
@@ -124,12 +177,33 @@ class JdbcReaderOptions:
 
 ### column_mapping
 
-```python
+<table>
+<tr>
+<th>python</th>
+<th>json</th>
+</tr>
+<tr>
+<td>
+<pre lang="python">
+
 @dataclass
 class ColumnMapping:
     source_name: str
     target_name: str
-```
+</pre>
+</td>
+<td>
+<pre lang="json">
+"column_mapping":[
+  {
+    "source_name": "p_id",
```

Comment:
Please update all the actual values with <TO-BE-FILLED> elements.

Reply:
Fixed